schemas/test/en/test_date_validation_range.json

```diff
@@ -47,7 +47,7 @@
                                 "guidance": {
                                     "contents": [
                                         {
-                                            "list": ["Enter a date range between 23 days and 1 month and 20 days apart"]
+                                            "description": "Enter a date range between 23 days and 1 month and 20 days apart"
```

Comment:
Unnecessary change committed?

Reply:
No I noticed that in one of the test schemas the guidance was wrongly formatted so fixed it